app/javascript/components/modals/index.ts

```diff
@@ -1,2 +1,3 @@
 export { FirstTimeModal } from './FirstTimeModal'
 export { Modal } from './Modal'
+export { UploadVideoModal } from './upload-video'
```

Comment:
this is stored among modals, but it is not a general modal, as it uses the context of the approaches page. 
- maybe move it from here?
- rework so it has an inner context
- leave it as it is, as modals folder is just for...any modal?

Reply:
I'm fine with it being here for now.